app/models/master_file.rb

```diff
@@ -308,6 +308,13 @@ def update_progress_on_success!(encode)
     # is stored as an integer string
     self.duration = encode.input.duration.to_i.to_s if encode.input.duration.present?
 
+    # Input videos that are in portrait orientation can have metadata showing landscape orientation
+    # with a rotation value. This can cause the aspect ratio on the master file to be incorrect. 
+    # We can get the proper aspect ratio from the transcoded files, so we set the master file off the 
+    # encode output.
+    high_output = Array(encode.output).select { |out| out.label.include?("high") }.first
+    self.display_aspect_ratio = (high_output.width.to_f / high_output.height.to_f).to_s
```

Comment:
This looks good.  I think it just needs to be in a check for video.
```suggestion
    if is_video?
      high_output = Array(encode.output).select { |out| out.label.include?("high") }.first
      self.display_aspect_ratio = (high_output.width.to_f / high_output.height.to_f).to_s
    end
```

Reply:
Yep, good catch :100: